File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/gpt-neo-2.7/knapsack.py
--- 
+++ 
@@ -7,7 +7,7 @@
         for j in range(1, capacity + 1):
             memo[i, j] = memo[i - 1, j]
 
-            if  weight < j:
+            if j >= weight:
                 memo[i, j] = max(
                     memo[i, j],
                     value + memo[i - 1, j - weight]

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/incoder-1/knapsack.java
--- 
+++ 
@@ -14,7 +14,7 @@
             if (i == 0 || j == 0) {
                 memo[i][j] = 0;
             }
-            else if (weight < j) {
+            else if (j - weight >= 0) {
                 memo[i][j] = Math.max(memo[i - 1][j], value + memo[i - 1][j - weight]);
             }
             else {
